core/config/constants.js

```diff
@@ -106,6 +106,12 @@ const defaultSettings = {
 
   auditMode: false,
   gatherMode: false,
+  /** Clear some types of storage.
+   Cookies are not cleared, so the user isn't logged out.
+  indexeddb, websql, and localstorage are not cleared to prevent loss of potentially important data.
+  https://chromedevtools.github.io/debugger-protocol-viewer/tot/Storage/#type-StorageType
+  */
```

Comment:
I don't think many people are going to see this comment. It might be better to mention the defaults in the JSDOC comment in `settings.d.ts` where IDEs have access to it.